### PR TITLE
Add cloud specific routing

### PR DESCRIPTION
**Pull request by karelhala**
This PR will enable routing on cloud server and make dynaamic routing. URL you want to access is `/hcm/settings/rbac`

### CODE REVIEW

**Comment by karelhala** [comment]
ping @gmcculloug

File: config/spandx.config.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/*global module, process*/
+
+// Hack so that Mac OSX docker can sub in host.docker.internal instead of localhost
+// see https://docs.docker.com/docker-for-mac/networking/#i-want-to-connect-from-a-container-to-a-service-on-the-host
+const localhost = (process.env.PLATFORM === 'linux') ? 'localhost' : 'host.docker.internal';
```
Comment (gmcculloug) [inline config/spandx.config.js]: I did not think we needed this line anymore.  I recall some email on Ian on it.

File: package.json [hunk 2]
```diff
@@ -120,6 +120,6 @@
     "verify": "npm-run-all build lint test"
   },
   "insights": {
-    "appname": "insights-rbac"
+    "appname": "rbac"
```
Comment (Fryguy) [inline package.json]: Does line 2 have to change also?  I've been trying to keep both of these lines in sync.

Comment (karelhala) [reply package.json]: Not really. The second line in this file represents package name, this on the other hand is representing application name in the insights/cloud services project.

**Pull request merged**